image.yaml

```diff
@@ -1,16 +1,16 @@
 schema_version: 1
 
-name: "jboss-eap-7-tech-preview/eap-cd-openshift"
-description: "The JBoss EAP continuous delivery (JBoss EAP CD) releases are intended to be Technology Preview - The JBoss EAP CD OpenShift container image is provided as technology preview. It is intended for development use only. It should NOT be deployed on production or in environments that are not intended for development use."
-version: "13.0"
-from: "jboss-eap-7/eap-cd:13.0"
+name: "jboss-eap-7-tech-preview/eap72-openshift"
```

Comment:
OK, final note on this.

When we were going to use -beta, Marek pointed out that cekit / concreate didn't support adding it to the image name, but now we're back using tech-preview, I think we should set this to

name: "jboss-eap-7/eap72-openshift"

Then build using concreate / cekit with --build-tech-preview.

These are related to changes that have been in the works regarding unit test annotations etc and EAP CD (which has -tech-preview in the prefix too. This will make it much easier to add tests as we go etc.

Reply:
Just to complete this thought, we should set the name in image.yaml to:

name: "jboss-eap-7/eap72-openshift"

But then override it in all of the -override.yaml files to:

name: "jboss-eap-7-tech-preview/eap72-openshift"

This change can also wait until the next sprint if we want, as we haven't updated cct_module tests etc quite yet.